### PR TITLE
Bug 2128074: Fix last sync time for DR

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
Screenshot from 2022-09-19 23-52-43

File: packages/shared/details-page/datetime.ts [hunk 1]
```diff
@@ -231,3 +231,8 @@ export const twentyFourHourTime = (
   const seconds = showSeconds ? `:${zeroPad(date.getSeconds() ?? 0)}` : '';
   return `${hours}${minutes}${seconds}`;
 };
+
+export const getLatestDate = (dates: string[]) =>
+  dates.reduce((a, b) => {
+    return new Date(a) > new Date(b) ? a : b;
+  });
```
Comment (SanjalKatiyar) [inline packages/shared/details-page/datetime.ts:238]: nits:
```suggestion
  dates.reduce((acc, date) =>  new Date(acc) > new Date(date) ? acc : date);
```

File: packages/mco/components/disaster-recovery/app-dr-status/dr-status-table.tsx [hunk 2]
```diff
@@ -1,4 +1,9 @@
 import * as React from 'react';
+import {
+  getLatestDate,
+  utcDateTimeFormatterWithTimZone,
```
Comment (SanjalKatiyar) [inline packages/mco/components/disaster-recovery/app-dr-status/dr-status-table.tsx:4]: not related to this PR, we can skip and fix later as well if we want...
```suggestion
  utcDateTimeFormatterWithTimeZone,
```

File: packages/mco/components/disaster-recovery/app-dr-status/dr-status-table.tsx [hunk 3]
```diff
@@ -98,9 +106,17 @@ const getDRStatus = (
   }
 };
 
+const getSyncStatus = (syncTime: string, t: TFunction) =>
```
Comment (SanjalKatiyar) [inline packages/mco/components/disaster-recovery/app-dr-status/dr-status-table.tsx:109]:
```suggestion
const getSyncStatus = (syncTime: string) =>
```

File: packages/mco/components/disaster-recovery/app-dr-status/dr-status-table.tsx [hunk 4]
```diff
@@ -135,10 +151,10 @@ export const DRPoliciesStatusTable: React.FC<DRPoliciesStatusTableProps> = ({
       _.map(drPolicies, (drpcList, policyName) => {
         const currentStatus = getCurrentStatus(drpcList);
         const targetClusters = getTargetClusters(currentStatus, drpcList);
+        const lastSyncTime = getLastDataSyncTime(drpcList);
         return {
           drPolicyName: policyName,
-          // TODO:Gowtham Once it is available on DRPC CR
-          lastSync: '',
+          lastSync: getSyncStatus(lastSyncTime, t),
```
Comment (SanjalKatiyar) [inline packages/mco/components/disaster-recovery/app-dr-status/dr-status-table.tsx:157]:
```suggestion
          lastSync: getSyncStatus(lastSyncTime),
```

**Comment by GowthamShanmugam** [comment]
@SanjalKatiyar Changes done

**Pull request merged**